webapp/src/Controller/API/AccessController.php

```diff
@@ -24,6 +24,7 @@ class AccessController extends AbstractRestController
     /**
      * Get access information
      *
+     * @return array<string, array<int, array<string, array<int, string>|string>|string>>
```

Comment:
We should either:

* Use an array shape
* Change this method to return a DTO (data transfer object) instead of an array

I'm leaning towards the second option